app/src/org/commcare/android/view/SquareImageView.java

```diff
@@ -0,0 +1,37 @@
+package org.commcare.android.view;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+
+/**
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+public class SquareImageView extends ImageView {
+
+    public SquareImageView(Context context) {
+        super(context);
+    }
+
+    public SquareImageView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public SquareImageView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    @Override
+    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
```

Comment:
Copied this code from `SquareButton`, don't exactly understand the scaling logic. Will defer to @ctsims.